src/js/services/forms.js

```diff
@@ -10,6 +10,16 @@ import { FORM_RESPONSE_STATUS } from 'js/static';
 
 import { versions } from 'js/config';
 
+function getClinicians(teamId) {
+  if (teamId) {
+    const team = Radio.request('entities', 'teams:model', teamId);
+    return team.getAssignableClinicians();
```

Comment:
It's not possible for the same `teamId` to exist across workspaces?

Reply:
all requests on the FE are filtered to a single workspace, so there's no teams that are not in the currently selected workspace at this point.